src/pages/NftCollectionInfo/NftCollectionOverview/index.tsx

```diff
@@ -7,6 +7,7 @@ import { explorerService } from '../../../services/ExplorerService'
 import styles from './styles.module.scss'
 import { handleNftImgError, patchMibaoImg } from '../../../utils/util'
 import { getPrimaryColor } from '../../../constants/common'
+import XUDTTag from '../../../components/XUDTTag'
```

Comment:
Why use `xudt` tag in `nft` component

Reply:
> Why use `xudt` tag in `nft` component

They are using the same style, so I just reuse it

Reply:
They are semantically different so it's better not to reuse the code even now they are the same.